src/lib/serviceRoutines/postSubscribeContext.cpp

```diff
@@ -51,6 +52,7 @@
 *
 * URI parameters
 *   - notifyFormat=XXX    (used by mongoBackend)
```

Comment:
notifyFormat is no longer used after removing XML. However, not so important for this PR. It can be removed within the "Remove XML (part X)" serie.

NTC